src/main/java/ca/fxco/pistonlib/blocks/PoweredStickyBlock.java

```diff
@@ -47,7 +47,7 @@ protected void createBlockStateDefinition(StateDefinition.Builder<Block, BlockSt
 
     @Override
     public void onRemove(BlockState state, Level level, BlockPos pos, BlockState newState, boolean movedByPiston) {
-        if (!level.isClientSide() && !newState.is(this) && movedByPiston && state.getValue(POWERED)) {
+        if (!level.isClientSide() && !newState.is(this) && !movedByPiston && state.getValue(POWERED)) {
```

Comment:
Why does this block update its powered state upon being removed anyway? Shouldn't its `onRemove` implementation just be left out entirely?

Reply:
ill check in a bit

Reply:
yup not needed